eigrpd/eigrp_interface.c

```diff
@@ -91,7 +85,7 @@ struct eigrp_interface *eigrp_if_new(struct eigrp *eigrp, struct interface *ifp,
 	ei->curr_bandwidth = ifp->bandwidth;
 	ei->curr_mtu = ifp->mtu;
 
-	return ei;
+	return 0;
```

Comment:
If this is just returning, we might then use a void() at all?

Reply:
It is a hook definition; the declaration is expected to return int. The same goes for interface deletion.